core/templates/pages/admin-page/navbar/admin-navbar.directive.html

```diff
@@ -48,166 +48,175 @@
+            <li class="nav-item oppia-clickable-navbar-element float-right">
+              <a class="nav-link oppia-navbar-tab"
+                 ng-href="<[::$ctrl.ADMIN_TAB_URLS.ACTIVITIES]>"
+                 ng-class="{active: $ctrl.isActivitiesTabOpen()}"
+                 uib-tooltip="Activities"
+                 tooltip-placement="bottom">
+                Activities
+              </a>
+            </li>
+            <li class="nav-item oppia-clickable-navbar-element float-right">
+              <a class="nav-link oppia-navbar-tab"
+                 ng-href="<[::$ctrl.ADMIN_TAB_URLS.JOBS]>"
+                 ng-class="{active: $ctrl.isJobsTabOpen()}"
+                 uib-tooltip="Jobs"
+                 tooltip-placement="bottom">
+                Jobs
+              </a>
+            </li>
+            <li class="nav-item oppia-clickable-navbar-element float-right">
+              <a class="nav-link oppia-navbar-tab protractor-test-admin-config-tab"
+                 ng-href="<[::$ctrl.ADMIN_TAB_URLS.CONFIG]>"
+                 ng-class="{active: $ctrl.isConfigTabOpen()}"
+                 uib-tooltip="Config"
+                 tooltip-placement="bottom">
+                Config
+              </a>
+            </li>
+            <li class="nav-item oppia-clickable-navbar-element float-right">
+              <a class="nav-link oppia-navbar-tab protractor-test-admin-roles-tab"
+                 ng-href="<[::$ctrl.ADMIN_TAB_URLS.ROLES]>"
+                 ng-class="{active: $ctrl.isRolesTabOpen()}"
+                 uib-tooltip="Roles"
+                 tooltip-placement="bottom">
+                Roles
+              </a>
+            </li>
+            <li class="nav-item oppia-clickable-navbar-element float-right">
+              <a class="nav-link oppia-navbar-tab"
+                 ng-href="<[::$ctrl.ADMIN_TAB_URLS.MISC]>"
+                 ng-class="{active: $ctrl.isMiscTabOpen()}"
+                 uib-tooltip="Miscellaneous"
+                 tooltip-placement="bottom">
+                Misc
+              </a>
+            </li>
+          </ul>
+        </li>
+        <li>
+          <ul class="nav oppia-navbar-tabs-admin">
+            <li uib-dropdown class="nav-item uib-dropdown float-right">
+              <a uib-dropdown-toggle class="nav-link uib-dropdown-toggle oppia-navbar-dropdown-toggle"
+                 data-toggle="dropdown"
+                 ng-mouseover="$ctrl.onMouseoverProfilePictureOrDropdown($event)"
+                 ng-mouseleave="$ctrl.onMouseoutProfilePictureOrDropdown($event)">
+                <div class="oppia-navbar-profile-picture-container float-left" ng-cloak>
+                  <div ng-if="$ctrl.profilePictureDataUrl">
+                    <img ng-src="<[$ctrl.profilePictureDataUrl]>"
+                         class="dropdown-toggle oppia-navbar-profile-picture rounded-circle"
+                         alt = "">
```

Comment:
Why is the alt an empty string here? Also remove extra whitespace.

Reply:
So I wasn't sure if describing this image would be useful to the user. And according the audit documentation, 

>  If the image acts as decoration and does not provide any useful content, give it an empty alt="" attribute to remove it from the accessibility tree:

https://web.dev/image-alt/?utm_source=lighthouse&utm_medium=devtools

Reply:
Done, White space removed